spec/epochs_spec.lua

```diff
@@ -684,14 +684,15 @@ describe("epochs", function()
 			}
 
 			local epochIndex = 0
+			local originalDelegatedStake = 100000000
 			for i = 1, 5 do
 				local gateway = {
 					operatorStake = gar.getSettings().operators.minStake,
-					totalDelegatedStake = 100000000,
+					totalDelegatedStake = originalDelegatedStake,
```

Comment:
As a reader and test debugger this past week, I've found the use of variables in tests for testing values to be a detractor from readability, SEO, and debuggability. In the past I've found small comments accompanying hardcoded values to provide context as a useful way to go instead.

Using variables in this way starts to lean towards writing too much logic into TESTING code that itself could be faulty or hide problems when conditions dynamically change in the surrounding test code. I prefer the security, SEO, and debuggability provided by rigid constants.

Reply:
fair - this one is used 7 times in this test align to compute expected rewards, but will adjust